settings.lua

```diff
@@ -2227,13 +2344,14 @@ end
 
 ---@class StopAtEndSetting : BooleanSetting
 StopAtEndSetting = CpObject(BooleanSetting)
-function StopAtEndSetting:init(vehicle)
+function StopAtEndSetting:init(vehicle,driverModeSetting)
```

Comment:
(this is really not important but a space after the comma would help readability)